Applications/Basic/CaptureHDRLoop/CaptureHDRLoop.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+This example shows how to acquire HDR images from the Zivid camera in a loop
+(while actively changing some HDR settings). Two methods are demonstrated:
+the direct HDR capture, which is faster, and the indirect method, which is
+slower but preserves single captures.
+*/
+
+#include <Zivid/CloudVisualizer.h>
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Setting up visualization" << std::endl;
+        Zivid::CloudVisualizer vis;
+        zivid.setDefaultComputeDevice(vis.computeDevice());
+
+        std::cout << "Connecting to the camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        std::cout << "Acquiring HDR frame directly (faster)" << std::endl;
+        std::vector<Zivid::Settings> settingsVector;
+
+        size_t iris[3] = { 10U, 20U, 30U };
+        int exposureTime[3] = { 10000, 20000, 30000 };
+
+        for(size_t i = 0; i < 3; ++i)
+        {
+            settingsVector.push_back(
+                Zivid::Settings{}
+                    .set(Zivid::Settings::Iris{ iris[i] })
+                    .set(Zivid::Settings::ExposureTime{ std::chrono::microseconds{ exposureTime[i] } })
+                    .set(Zivid::Settings::Brightness{ 1 })
+                    .set(Zivid::Settings::Gain{ 1 })
+                    .set(Zivid::Settings::Bidirectional{ false })
+                    .set(Zivid::Settings::Filters::Contrast::Enabled::yes)
+                    .set(Zivid::Settings::Filters::Contrast::Threshold{ 5 })
+                    .set(Zivid::Settings::Filters::Gaussian::Enabled::yes)
+                    .set(Zivid::Settings::Filters::Gaussian::Sigma{ 1.5 })
+                    .set(Zivid::Settings::Filters::Outlier::Enabled::yes)
+                    .set(Zivid::Settings::Filters::Outlier::Threshold{ 5 })
+                    .set(Zivid::Settings::Filters::Reflection::Enabled::yes)
+                    .set(Zivid::Settings::Filters::Saturated::Enabled::yes)
+                    .set(Zivid::Settings::BlueBalance{ 1.081 })
+                    .set(Zivid::Settings::RedBalance{ 1.709 }));
+            std::cout << "Frame " << i << " " << settingsVector[i] << std::endl;
+        }
+
+        auto hdrFrame = Zivid::HDR::capture(camera, settingsVector);
+
+        std::cout << "Acquiring HDR source images and combining them into HDR frame (slower)" << std::endl;
+        std::vector<Zivid::Frame> frames;
+
+        for(size_t i = 0; i < 3; ++i)
+        {
+            camera << Zivid::Settings::Iris{ iris[i] }
```

Comment:
Why did you move all these settings inside the loop? I think it was better before when all the non-changing settings were applied once.

Reply:
Done